python/glow/gwas/log_reg.py

```diff
@@ -38,15 +38,16 @@ def logistic_regression(genotype_df: DataFrame,
     On the driver node, we fit a logistic regression model based on the covariates for each
```

Comment:
Add "Implementation details" to before this paragraph like what we have for lin_reg